plugins/scaffolder/api-report.md

```diff
@@ -235,6 +235,7 @@ export const MultiEntityPickerFieldExtension: FieldExtensionComponent_2<
         >[]
       | undefined;
     allowArbitraryValues?: boolean | undefined;
+    maxNoOfEntities?: number | undefined;
```

Comment:
Ah ok, yep `maxItems` I think is what we want to use right instead? You can access the raw jsonschema with `schema` I think in the props.

Take this for example:

```yaml
entity:
  title: Entities
  type: array
  description: Entities of the component
  maxItems: 3
  ui:field: MultiEntityPicker
```

Instead of using a `ui:option` for this we'd just rely on the `maxItems` that's part of the `jsonschema` spec?